<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_chk">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>150</y>
      <width>75</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>btn_chk</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_insert">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>110</y>
      <width>75</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>btn_insert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remove">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>75</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>btn_remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>210</y>
      <width>75</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>btn_1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>240</y>
      <width>75</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>btn_3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_enable">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>75</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>btn_enable</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
